CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+### v5.0.0-rc.2 (2024-04-22)
+
+- Use visibility-state performance entries ([#612](https://github.com/GoogleChrome/web-vitals/pull/612))
+- Add upgrading to v5 doc ([#558](https://github.com/GoogleChrome/web-vitals/pull/558))
```

Comment:
Nit: I think it would be fine to omit this from the CHANGELOG (or not, either way)

Reply:
Figured no harm to give this extra visibility. Especially for RC where we won't be announcing it as much.

In fact for that reason, I've moved it to be the first PR noted here in https://github.com/GoogleChrome/web-vitals/pull/613/commits/e6c8b48c99f249ef0b4e741352dfa59ac4c24879